cms/static/js/views/course_highlights_enable.js

```diff
@@ -0,0 +1,53 @@
+define([
+    'jquery', 'underscore', 'backbone', 'js/views/utils/xblock_utils', 'js/utils/templates',
+    'js/views/modals/course_outline_modals'],
+    function(
+        $, _, Backbone, XBlockViewUtils, TemplateUtils, CourseOutlineModalsFactory
+    ) {
+        'use strict';
+        var CourseHighlightsEnableView = Backbone.View.extend({
+            events: {
+                'click button.status-highlights-enabled-value': 'handleEnableButtonPress',
+                'keypress button.status-highlights-enabled-value': 'handleEnableButtonPress'
+            },
+
+            initialize: function() {
+                this.template = TemplateUtils.loadTemplate('course-highlights-enable');
+            },
+
+            handleEnableButtonPress: function(event) {
+                if (event.type === 'click' || event.which === 13 || event.which === 32) {
```

Comment:
What do 13 and 32 mean here? Can we name these numbers as variables?

Reply:
It's enter and space. I'm annoyed by these numbers too. I looked it up and I think we could use [KeyboardEvent.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead which gives a string name instead of an int. I'll make this change.